openlibrary/components/BulkSearch/components/BulkSearchControls.vue

```diff
@@ -8,17 +8,14 @@
             <textarea v-model="bulkSearchState.inputText"></textarea>
             <br />
             <label>Format: <select @change="selectAlgorithm">
-                    <option value="1">e.g. "The Wizard of Oz by L. Frank Baum"</option>
-                    <option value="2">e.g. "L. Frank Baum - The Wizard of Oz" </option>
-                    <option value="3">e.g. "The Wizard of Oz - L. Frank Baum" </option>
-                    <option value="4">e.g. "The Wizard of Oz (L. Frank Baum)"</option>
-                    <option value="5">Wikipedia Citation (e.g. Baum, Frank L. (1994). The Wizard of Oz)</option>
-                    <option value="6">✨ AI Extraction</option>
+                    <option v-for="extractor, index in extractors" :value = "index" :key="index">
+                        {{ extractor["name"] }}
+                    </option>
                 </select></label>
-            <label v-if="bulkSearchState.extractionOptions.use_gpt">OpenAI API Key:
-                <input v-if="showPassword" type="password" @click="togglePasswordVisibility()" v-model="bulkSearchState.extractionOptions.api_key" />
+            <label v-if="this.showApiKey">OpenAI API Key:
+                <input v-if="showPassword" type="password" v-on:click="togglePasswordVisibility()" v-model="bulkSearchState.extractionOptions.api_key" />
```

Comment:
Oh this might've been an auto-formater, but we prefer the `@click` syntax! Update this here and elsewhere below.

```suggestion
                <input v-if="showPassword" type="password" click="togglePasswordVisibility()" v-model="bulkSearchState.extractionOptions.api_key" />
```

Reply:
Yeah, I was getting some error that I can't recall at the moment. I've changed this back, however, and it's working fine, so I guess no need to care.